style.css

```diff
@@ -0,0 +1,360 @@
+  color: #fff;
+}
+
+/* End Header Section */
+
+/* Hero Section */
+
+#hero {
+  height: 18rem;
+  background-image: url('https://upload.wikimedia.org/wikipedia/commons/thumb/4/40/Ambrose_Akinmusire_2011.jpg/1200px-Ambrose_Akinmusire_2011.jpg');
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-attachment: fixed;
+  box-shadow: inset 0 0 0 1000px rgba(255, 255, 255, 0.11);
+  margin-top: 3.5rem;
+  content: '';
+  z-index: -1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  color: white;
+  left: 0;
+  right: 0;
+  position: relative;
+  width: 100%;
+}
+
+#img-title {
+  margin-top: -15px;
+}
+
+#hero hr {
+  width: 70%;
+  height: 2px;
+  border-width: 0;
+  color: white;
+  background-color: white;
+  margin-top: -10px;
+  margin-bottom: 30px;
+}
+
+#hero p {
+  font-size: 1.6rem;
+  margin-top: -10px;
+}
+
+#form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-around;
+}
+
+#email {
+  margin-top: -5px;
+  height: 2rem;
+  width: 300px;
+  text-align: center;
+  border: 2px solid #000;
+  border-radius: 4px;
+  font-size: 1.1rem;
+}
+
+#submit {
+  margin-top: 5px;
+  height: 2rem;
+  width: 150px;
+  transition: color 0.6s ease, background-color 0.6s ease;
+  background-color: var(--back-color);
+  color: #000;
+  border: 2px solid #000;
+  border-radius: 4px;
+  font-size: 1rem;
+  text-align: center;
+  margin-top: 10px;
+  font-weight: 700;
+}
+
+#submit:hover {
+  background-color: #000;
+  color: var(--back-color);
+}
+
+/* End Hero Section */
+
+/* Main Section */
+
+#features {
+  position: relative;
+  margin: 3rem 45rem 2rem 45rem;
+  z-index: -1000;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 20rem;
+}
+
+.container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  width: 800px;
+}
+
+#icon1 {
+  height: 60px;
+  width: auto;
+  margin-top: 25px;
+}
+
+#icon2 {
+  height: 60px;
+  width: auto;
+  margin-top: 20px;
+}
+
+#icon3 {
+  height: 60px;
+  width: auto;
+  margin-top: 30px;
+}
+
+.info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+}
+
+.info p {
+  margin-top: -20px;
+}
+
+#features .fhr {
+  color: #000;
+  background-color: #000;
+  width: 800px;
+  height: 5px;
+  border-width: 0;
+}
+
+/* video section */
+.video {
+  margin-top: 3.125rem;
+  display: flex;
+  justify-content: center;
+  width: 100%;
+}
+
+.video iframe {
+  max-width: 35rem;
+  width: 100%;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  height: 12.5rem;
+  margin-top: 3.125rem;
+}
+
+@media (max-width: 1200px) {
+  footer {
+    background-color: #f3f3f3;
+  }
+}
+
+.form form {
+  max-width: 17.1875rem;
+  width: 100%;
+  padding: 0.3125rem;
+}
+
+.form p {
```

Comment:
Expected selector ".info p" to come before selector "#hero p"

Reply:
- [ ] This code snippet does not indicate the portion of code to change. It would be nice to show the exact place in the code where a change is requested of a developer. This will avoid misunderstandings or assumptions and will give the developer clarity.